app/views/courses/_users.json.jbuilder

```diff
@@ -4,7 +4,9 @@ json.users course.courses_users.eager_load(:user) do |cu|
   json.call(cu.user, :id, :wiki_id, :contribution_url, :sandbox_url)
   json.admin cu.user.permissions == User::Permissions::ADMIN
   json.recent_revisions RevisionStat.recent_revisions_for_user_and_course(cu.user, cu.course).count
-  json.course_training_progress ctp_manager.course_training_progress
+  if ENV['disable_training'] != 'true'
```

Comment:
The `unless` form would be more consistent with the rest of the codebase. We try to follow the ruby style guide (https://github.com/bbatsov/ruby-style-guide#unless-for-negatives) as much as possible.